site/public/js/server.js

```diff
@@ -1375,7 +1375,11 @@ function peerFeedbackUpload(grader_id, user_id, g_id, feedback){
         contentType: false,
         success: function(data) {
             try {
```

Comment:
Is the try catch needed here, I don't think the `$.html` function throws any errors and if the `$.getJSON` function fails it would hit the `error` function